src/persistence/entities/applicationData.entity.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const BaseModel = require('./base.entity')
```

Comment:
Should refer to this as BaseEntity, just to avoid confusion.

Reply:
Changed to BaseEntity where required